app/controllers/mixins/actions/host_actions/misc.rb

```diff
@@ -123,17 +123,6 @@ def process_hosts_introspect(hosts, display_name)
           end
         end
 
-        def process_host_provide(hosts, display_name)
```

Comment:
I think this is dead due to a typo in https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/mixins/actions/host_actions/misc.rb#L163

So we should fix that rathe that remove.

Reply:
@doruskova I took the liberty of updating your PR to rename this method to `process_hosts_provide` instead.